src/components/UserForm.tsx

```diff
@@ -1,17 +1,18 @@
 import React, { useState, useContext } from 'react';
 import { Container, Title, TextInput, Button, Group, Header } from '@mantine/core';
 import { useStateWithLocalStorage } from './UseStateWithLocalStorage';
-import { WeatherContext } from './WeatherComponent';
 import { MdWbSunny } from 'react-icons/md';
 import { BsFillCloudSnowFill } from 'react-icons/bs';
 import { IoIosPartlySunny } from 'react-icons/io';
 import { useStyles } from '../styles/Styles';
+import { WeatherContext } from './context/WeatherContext';
 
 const UserForm = () => {
   const [inputValue, setInputValue] = useStateWithLocalStorage('', 'form');
   const [show, setShow] = useState(true);
   const { classes } = useStyles();
   const weatherIcon = useContext(WeatherContext);
```

Comment:
Same thing in this spot, deconstruct the object so you can just access the wether data directly